cardboard/settings.py

```diff
@@ -172,7 +177,7 @@
 # Configure Django App for Heroku.
 import django_heroku
 
-django_heroku.settings(locals(), test_runner=False)
+django_heroku.settings(locals(), test_runner=False, staticfiles=False)
```

Comment:
Should we add a comment above this line pointing out that `staticfiles=False` is to avoid clobbering previous settings?

Reply:
Where did this change come from?